### PR TITLE
[VL] Enable decimalV2A test

### PR DESCRIPTION
**Pull request by JkSelf**
## What changes were proposed in this pull request?

(Please fill in changes proposed in this fix)

## How was this patch tested?

(Please explain how this patch was tested. E.g. unit tests, integration tests, manual tests)

(If this patch involves UI changes, please attach a screenshot; otherwise, remove this)

### CODE REVIEW

File: cpp/velox/tests/ArrowToVeloxTest.cc [hunk 1]
```diff
@@ -118,10 +118,10 @@ TEST_F(ArrowToVeloxTest, decimalV2A) {
   {
     // only RowVector can convert to RecordBatch
     auto row = makeRowVector({
-        makeShortDecimalFlatVector({1000265, -35610, 0}, DECIMAL(10, 3)),
+        makeShortDecimalFlatVector({1000265000, -35610000, 0}, DECIMAL(10, 3)),
```
Comment (JkSelf) [inline cpp/velox/tests/ArrowToVeloxTest.cc:121]: @jinchengchenghh
 The RowVector store the [UnscaledShortDecimal ](https://github.com/facebookincubator/velox/blob/94644c62f34daf966d32dadb5a0e059530eeaff9/velox/type/UnscaledShortDecimal.h#L27) value and we need change the value from 1000265, -35610, 0 to 1000265000, -35610000, 0.

**Review by jinchengchenghh — APPROVED** [review]

**Comment by jinchengchenghh** [comment]
Can you rebase and rerun the test?

**Comment by kyligence-git** [comment]
Can one of the admins verify this patch?

**Comment by JkSelf** [comment]
> Can you rebase and rerun the test?

Done.

**Comment by JkSelf** [comment]
@jinchengchenghh Please help to review again. Thanks.

**Review by jinchengchenghh — APPROVED** [review]

**Pull request merged**